studio/src/ui/popoverMenus/ContextMenuItem.js

```diff
@@ -17,22 +23,36 @@ export class ContextMenuItem {
 		icon = null,
 	} = {}) {
 		this.containingContextMenu = containingContextMenu;
-		this.el = document.createElement("div");
-		this.el.classList.add("contextMenuItem");
-		this.el.classList.toggle("disabled", disabled || horizontalLine);
+
+		if (horizontalLine) {
+			this.el = document.createElement("div");
```

Comment:
`<hr>` is a void element so it can't accept any inner tags. Removing the early return statement means that the rest of the elements are added, so I changed back to a `<div>`.

We could break out the `ContextMenuItem` into a "MenuItem" and "MenuDivider", so we can keep the nice semantics while keeping type checking more sane.

Reply:
This sounds nice, as long as it doesn't impact the way structures are defined too much.
I.e. ideally this would still work:
```js
const menu = getStudioInstance().popoverManager.createContextMenu([
	{
		text: "a",
	},
	{
		horizontalLine: true,
	},
	{
		text: "b",
	}
]);
```

But either way I think changing this is pretty low priority. The macOS voice assistent didn't announce the horizontal line anyway iirc, even when it was still a `<hr>`.